samples/dotnet/KernelHttpServer/Extensions.cs

```diff
@@ -132,7 +132,7 @@ internal static void RegisterNativeGraphSkills(this IKernel kernel, string graph
 
     internal static void RegisterTextMemory(this IKernel kernel)
     {
-        _ = kernel.ImportSkill(new TextMemorySkill(), nameof(TextMemorySkill));
+        _ = kernel.ImportSkill(nameof(TextMemorySkill));
```

Comment:
This is just a string. Should be new TextMemorySkill, and pass in Memory.
See https://github.com/microsoft/semantic-kernel/pull/2207

Reply:
Fixed